tests/test_name_server.rb

```diff
@@ -65,10 +65,10 @@ def test_nameserver_create_destroy_single_ipv6
   end
 
   def test_router_create_destroy_multiple
-    id1 = '8.8.8.8'
-    id2 = '2001:4860:4860::8888'
+    id1 = '7.7.7.7'
+    id2 = '2001:4860:4860::7777'
+    refute_includes(Cisco::NameServer.nameservers, id1)
```

Comment:
@robert-w-gries I fixed this 

https://github.com/cisco/cisco-network-node-utils/commit/f2ddb8c74414484eb6a8b1cd479ee4c2d692adff